### DISCUSSION TITLE
File format for emissions inventory

### DISCUSSION THREAD

**Discussion by XIR-art** [uataq/stilt#122] [Users]
Hello!
I would like to ask you two questions
1、The emission inventory of wbb tutorial is a .rds file, is the file you are downloading an rds file, I am downloading an nc file, do I need to convert it to an rds file? How do I need to convert it, and how do I convert it please?
2. Is the concentration change value obtained after convolving the footprint file with the emission inventory the value of the latitude and longitude of the center of each grid?

Thank you for your reply and your time!
Have a nice life!

**Comment by tartanrunner25** [comment]
1) Either or is fine. When you start running your own test cases you will likely need to edit the convolution script so that it plays nice with the format of your data files. These scripts were designed to work with a simple test case, which may not work for your own simulations when you get to that point.

2) The emissions are cell centered.
